.github/workflows/test.yml

```diff
@@ -20,5 +20,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install pytest
           python -m pip install jsonschema
+      - name: Generate Test SQs
+        run: python sq-generator/sq_generator.py -i test/resources/testdata.json -o test/resources/generated-sqs/
```

Comment:
I wouldn't put the generated queries into the source folder `test`. Please use something like `target`.